src/pages/Campaign/CampaignButtonWithOptions.tsx

```diff
@@ -83,11 +83,27 @@ export default function CampaignButtonWithOptions({
       const encodedData = response.data.data.EncodedData
       try {
         await sendTransaction(rewardContractAddress, encodedData, BigNumber.from(0), transactionResponse => {
-          // TODO nguyenhuudungz: Compile a list of unclaimed rewards from `campaignLeaderboard`.
+          const accumulatedUnclaimedRewards = selectedCampaignLeaderboard.rewards
+            .filter(reward => !reward.claimed)
```

Comment:
we can combine **filter and reduce** => only reduce

Reply:
I prefer make it separated. Just my prefered.

Reply:
if **selectedCampaignLeaderboard** is not large, it is ok.

Reply:
yeah it's usually 1 or 2 items

Reply:
okay.